hoomd/md/pytest/test_methods.py

```diff
@@ -273,6 +285,92 @@ def test_npt_attributes_attached_3d(simulation_factory,
     npt.gamma = 2.0
     assert npt.gamma == 2.0
 
+    assert npt.translational_thermostat_dof == (0.0, 0.0)
+    npt.translational_thermostat_dof = (0.125, 0.5)
+    assert npt.translational_thermostat_dof == (0.125, 0.5)
+
+    assert npt.rotational_thermostat_dof == (0.0, 0.0)
+    npt.rotational_thermostat_dof = (0.5, 0.25)
+    assert npt.rotational_thermostat_dof == (0.5, 0.25)
+
+    assert npt.barostat_dof == (0.0, 0.0, 0.0, 0.0, 0.0, 0.0)
+    npt.barostat_dof = (1.0, 2.0, 4.0, 6.0, 8.0, 10.0)
+    assert npt.barostat_dof == (1.0, 2.0, 4.0, 6.0, 8.0, 10.0)
+
+
+def test_npt_thermalize_extra_dof(simulation_factory,
+                                  two_particle_snapshot_factory):
+    """Tests that NPT.thermalize_extra_dof can be called."""
+    all_ = hoomd.filter.All()
+    constant_t = hoomd.variant.Constant(2.0)
+    constant_s = [1, 2, 3, 0.125, 0.25, 0.5]
+    npt = hoomd.md.methods.NPT(filter=all_,
+                               kT=constant_t,
+                               tau=2.0,
+                               S=constant_s,
+                               tauS=2.0,
+                               box_dof=[True, True, True, True, True, True],
+                               couple='xyz')
+
+    sim = simulation_factory(two_particle_snapshot_factory())
+    sim.operations.integrator = hoomd.md.Integrator(0.005, methods=[npt])
+    sim.operations.schedule()
```

Comment:
This isn't necessary anymore, right? Or has the relevant PR not been merged yet? In which case maybe just need to keep this in mind when merging things together.

Reply:
Same applies to other calls to `schedule` below.

Reply:
Yes I should remove this now that we decided run(0) is the preferred solution.